### PR TITLE
Fix issues with dotnet quickstart container not building

### PR DESCRIPTION
**Pull request by bourgeoisor**
This PR fixes issues with the dotnet container discovered when CI checks were added in https://github.com/GoogleCloudPlatform/kubernetes-engine-samples/pull/370

It also bumps C# to 10.0 and .Net to 6.0

### CODE REVIEW

File: quickstart/dotnet/Program.cs [hunk 1]
```diff
@@ -1,27 +1,35 @@
...
+// License for the specific language governing permissions and limitations under
+// the License.
+
+// using System;
+// using Microsoft.AspNetCore.Builder;
+// using Microsoft.AspNetCore.Hosting;
+// using Microsoft.AspNetCore.Http;
+// using Microsoft.Extensions.DependencyInjection;
+// using Microsoft.Extensions.Hosting;
+// using Microsoft.AspNetCore.Mvc;
```
Comment (donmccasland) [inline quickstart/dotnet/Program.cs]: Why are these added, but commented out?

Comment (bourgeoisor) [reply quickstart/dotnet/Program.cs]: Juuust removed them while you were commenting -- they're stray, already gone :)

File: quickstart/dotnet/Startup.cs [hunk 2]
```diff
@@ -10,37 +10,38 @@
...
-    }
-}
+// the License.
+
+using System;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
```
[the remark this answers is not in the document]
Comment (bourgeoisor) [reply quickstart/dotnet/Startup.cs:20]: That's it for source, the big changes happened in the .csproj file
The GitHub diff that you see is because I changed from CRLF to LF line endings to be consistent with the rest of the repo.

**Review by donmccasland — APPROVED** [review]
